src/store.js

```diff
@@ -52,15 +53,32 @@ function getValueActive(activeKeys, key) {
   return activeKeys.has(key);
 }
 
-function getActiveKeys(activeKeys, prefix) {
+function getActiveKeys(activeKeys, previousKeys, prefix) {
   const result = [];
-  activeKeys.forEach(function(key) {
-    if (!prefix || strings.startsWith(key, prefix)) {
+  const previous = [];
+
+  function hasPrefix(key) {
+    return !prefix || strings.startsWith(key, prefix)
+  }
+
+  activeKeys.forEach(function (key) {
+    if (hasPrefix(key)) {
       result.push(key);
     }
   });
 
-  return result;
+  previousKeys.forEach(function (key) {
+    if (hasPrefix(key)) {
+      previous.push(key);
+    }
+  });
+
+  return {
+    current: result,
+    previous: previous,
+    added: setDifference(result, previous),
+    removed: setDifference(previous, result)
```

Comment:
added and removed don't seem to be used, can they just be removed?

Reply:
Yes, they can. They were used downstream but are not any longer.